src/main/java/org/cryptomator/macos/keychain/TouchIdKeychainAccess.java

```diff
@@ -0,0 +1,75 @@
+package org.cryptomator.macos.keychain;
+
+import org.cryptomator.integrations.common.OperatingSystem;
+import org.cryptomator.integrations.common.Priority;
+import org.cryptomator.integrations.keychain.KeychainAccessException;
+import org.cryptomator.integrations.keychain.KeychainAccessProvider;
+import org.cryptomator.macos.common.Localization;
+
+/**
+ * Stores passwords in the macOS system keychain. Requires an authenticated user to do so.
+ * Authentication is done via TouchID or password as a fallback, when TouchID is not available.
+ * <p>
+ * Items are stored in the default keychain with the service name <code>Cryptomator</code>, unless configured otherwise
+ * using the system property <code>cryptomator.integrationsMac.keychainServiceName</code>.
+ */
+@Priority(1000)
```

Comment:
I'd say, Touch ID has a higher Priority than Non-Touch ID. *BUT*
* the `isSupported` method must properly detect, if touch ID is set up/enabled
* the old, non-biometric-protected data is migrated

Reply:
> I'd say, Touch ID has a higher Priority than Non-Touch ID. _BUT_
> 
> * the `isSupported` method must properly detect, if touch ID is set up/enabled
> * the old, non-biometric-protected data is migrated

Both is implemented. Please not, due to the higher priority, on new installations, Touch ID will be set up as the KeychainAccessProvider.

Reply:
@tobihagemann Need your opinion here. I suggested to give the TouchID provider a higher prio, because it is more "modern". And as long as we only need to authenticate once per unlock, from my point of view, this is fine. On the plus side, the users will have a higher "security" feeling.

Reply:
I agree with giving the Touch ID provider a higher priority. 👍